hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -123,13 +127,15 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
             .setVolumeName(volumeName).setBucketName(bucketName);
 
     boolean deleteStatus = true;
+    long startNanos = Time.monotonicNowNanos();
+    String finalVolumeName = volumeName;
+    String finalBucketName = bucketName;
```

Comment:
Do you mean the following, as per the [previous comment](https://github.com/apache/ozone/pull/6321#issuecomment-1980611987) ?

```
final String finalVolumeName = volumeName;
final String finalBucketName = bucketName;
```

Reply:
Yes Tanvi, we need to make them final to use it with lambda function

Reply:
Yes @tanvipenumudy, we need to make them final to use it with lambda function